src/App.svelte

```diff
@@ -11,29 +11,56 @@
     events_store,
     ui_store,
     filter_toggles,
-    platform_config_store,
+    platform_config_store
   } from "./stores/store";
 
   // parameters
   const fetch_interval_ms = 30000;
 
   // variables
-  let mouse_xy = { x: 0, y: 0 };
-  let handleMouseMove = throttle((event) => {
+  const mouse_xy = { x: 0, y: 0 };
+  const handleMouseMove = throttle((event) => {
     mouse_xy.x = event.clientX;
     mouse_xy.y = event.clientY;
   }, 5);
 
+  const CONTENT_ANALYSIS_FIRST_COLUMN = 12;
+
   let width_mod_grid = Math.floor(document.body.clientWidth / 10) * 10;
   let height_mod_grid = Math.floor(document.body.clientHeight / 10) * 10;
 
-  let handleWindowResize = throttle(() => {
+  const handleWindowResize = throttle(() => {
     width_mod_grid = Math.floor(document.body.clientWidth / 10) * 10;
     height_mod_grid = Math.floor(document.body.clientHeight / 10) * 10;
   }, 500);
 
+  const string_to_boolean_or_value = v => v === "TRUE"
+    ? true
+    : v === "FALSE"
+    ? false
+    : v;
+
+  const get_content_analysis_object = (row, column_names, first_column) => column_names.slice(first_column)
+    .map((n, i) => ([n, row[first_column + i]]))
+    .reduce((a, [k, v]) => Object.assign(a, {
+      [k]: string_to_boolean_or_value(v)
+    }), { _column_names: column_names.slice(first_column) });
+
+  const content_analysis_to_row = (ca) => ca._column_names.map(n => ca[n]);
+
+  media_store.on("push", media => fetch(
+    `/.netlify/functions/googlesheets?request=updateRow` +
+      `&tab=${$platform_config_store["Title of tab with media assets"]}` +
+      `&row=${media.row}` +
+      `&offset=${CONTENT_ANALYSIS_FIRST_COLUMN}`, {
+      method: "POST",
+      body: JSON.stringify(content_analysis_to_row(
+        media.contentAnalysis))
+    }
+  ));
```

Comment:
this is another code path, but you may mean 'in case the push fails'
not sure we should, what will happen is that the data is going to be reset next time we pull from gdocs, that said we should probably try harder to push (and suspend pull while we do so) in case we hit an API use limit... not sure it's necesary for this patchset though.

Reply:
this is actuall why this code is here (and not say, in store.js) so that we can access the pull timeout function…

Reply:
Great, we'll see the API use limit problem in another PR then.